scripts/perf_build.sh

```diff
@@ -10,5 +10,38 @@ curl -SL https://codeload.github.com/Granulate/linux/zip/40a7823cf90a7e69ce8af88
 unzip -qq linux.zip
 rm linux.zip
 cd linux-*/
-make -C tools/perf LDFLAGS=-static
+
+# Aarch64's libc has no "PIC" version. libtraceevent has some dynamic plugins it's trying to build.
```

Comment:
Well it probably does have a PIC version, but at least, it's not coming installed by default, and I'm getting random errors on files from `libc.a` that should be recompiled with `-fPIC`. Elaborate